docs/sql-data-sources-avro.md

```diff
@@ -348,7 +348,7 @@ Configuration of Avro can be done via `spark.conf.set` or by running `SET key=va
     <td>snappy</td>
     <td>
       Compression codec used in writing of AVRO files. Supported codecs: uncompressed, deflate,
-      snappy, bzip2 and xz. Default codec is snappy.
+      snappy, bzip2, xz and zstandard. Default codec is snappy.
```

Comment:
I suggest removing this bit as it duplicates the information in the default column.

```suggestion
      snappy, bzip2, xz and zstandard.
```

Same here, of course: https://github.com/apache/spark/blob/bbeb8d7417bafa09ad5202347175a47b3217e27f/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L3604